### DISCUSSION TITLE
Long press on mobile

### DISCUSSION THREAD

**Discussion by blaize** [fyne-io/fyne#4064] [Q&A] [answered]
I can't find any event for long press on mobile.
Should I use TouchDown(*TouchEvent), set a timer to 500ms then call the long press event func if TouchUp(*TouchEvent) didn't fired before the timer finished ?
Is there a better way ?

**Comment by andydotxyz** [comment, marked as answer]
TappedSecondary is the high level event. Right click on desktop and long tap on mobile.

**Reply by blaize** [reply]
Thanks, that is exactly what I was looking for.
